src/Kraken.h

```diff
@@ -34,6 +34,7 @@ class Kraken {
    Kraken();
    Spear SetLocation(const std::string& location);
    void MaxWaitInMs(const int timeout);
+   void MaxChunkSizeInBytes(const size_t bytes);
```

Comment:
Please change this name,.. see also my other suggestions.   
If you do want a "setter" function instead of handling this through a Mock then the name definitely needs to be changed.